src/newrelic.js

```diff
@@ -86,4 +89,11 @@ exports.config = {
       '403',
     ],
   },
+  transaction_tracer: {
+    record_sql: 'raw',
+  },
+  slow_sql: {
+    enabled: true,
+    max_samples: 100,

```

Comment:
We should consider either explaining these or give links to the relevant docs. In any case, 100 queries per minute seems like something that we'd be tuning a lot. It might not be enough to find the queries of interest during high intensity while also producing really a lot of unnecessary data at other times. For example, I can imagine us usually leaving this at like 5, then temporarily bumping it up to 500 or something when investigating a particular issue that the 5 setting didn't find.